receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go

```diff
@@ -65,6 +63,74 @@ func (ghs *githubScraper) scrape(_ context.Context) (pmetric.Metrics, error) {
 	currentDate := time.Now().Day()
 	ghs.logger.Sugar().Debugf("current date: %v", currentDate)
 
-	ghs.logger.Sugar().Debug("github scraper has started")
+	// Enable the ability to override the endpoint for self-hosted github instances
```

Comment:
do you want to file an issue for this?

Reply:
I see the confusion, this comment reads odd. It's should say something like `The default endpoint for graphql is this but can be overridden through the basic endpoint configuration option and will be joined with api/graphql, for more information see this doc` or something to that affect. I'll adjust the comment to make it more clear.